specs/README.md

```diff
@@ -19,6 +19,7 @@ Here you'll find specifications for custom elements and other library features.
 | Combobox | -- | -- |
 | [Dialog](../packages/web-components/fast-foundation/src/dialog/dialog.spec.md) | :white_check_mark: | :white_check_mark: |
 | [Disclosure](../packages/web-components/fast-foundation/src/disclosure/disclosure.spec.md) | :white_check_mark: | -- |
+| [Flyout](./flyout.md) | :white_check_mark: | -- |
```

Comment:
I'll change the name of the file before merge to not outdate all the comments due to a file name change.

Reply:
Flyout may make this component more distinguishable from future popup/pop related components, so may still be the best option to make it distinct.